pkg/crc/preflight/preflight_checks_tray_windows.go

```diff
@@ -96,12 +96,12 @@ func fixTrayBinaryExists() error {
 		}
 	}
 	archivePath := filepath.Join(tmpArchivePath, filepath.Base(constants.GetCRCWindowsTrayDownloadURL()))
-	outputPath := constants.CrcBinDir
-	//err = goos.MkdirAll(filepath.Join(outputPath, "tray-windows"), 0750)
+	outputPath := constants.TrayBinaryDir
+	err = goos.MkdirAll(outputPath, 0750)
```

Comment:
Note for later: `extract.unzip()` is creating the destination directory, but `extract.untar()` does not do it. It would be a worthwhile improvement to do it in both cases, and not have to do it here.

Reply:
Actually looking at the code for `extract.unzip()` it is supposed to create the destination directory.. i am not passing the correct target then..  https://github.com/code-ready/crc/blob/1b0dda67aad99118d0688f151483b5cf51f22625/pkg/extract/extract.go#L130-L132